homeassistant/components/onewire/onewire_entities.py

```diff
@@ -86,12 +86,12 @@ def __init__(
         sensor_type: str,
         sensor_name: str,
         device_info: Dict[str, Any],
-        disable_startup: bool,
+        default_disabled: bool,
         owproxy: protocol._Proxy,
```

Comment:
This parameter should probably be renamed `ow_proxy` or even better `onewire_proxy`.